import-export-cli/cmd/init.go

```diff
@@ -327,5 +351,7 @@ func init() {
 		"YAML definition of API")
 	InitCommand.Flags().StringVarP(&initCmdSwaggerPath, "oas", "", "", "Provide an OpenAPI "+
 		"specification file for the API")
+	InitCommand.Flags().StringVar(&initCmdInitialState, "initial-state", "", "Provide the initial state "+
```

Comment:
@renuka-fernando Have we considered the possibility of using "lc-state"? IMO, "lc-state" is short. But "lc" it self wouldn't make any sense to a new user. If that's the case we can come up with something else?  I feel like we can come up with something better than "initial-state". Just a thought.